=== modified file 'src/global.h'
--- src/global.h	2008-06-30 15:54:08 +0000
+++ src/global.h	2008-07-05 17:14:43 +0000
@@ -158,7 +158,9 @@
 
 
 /// Handprint: CFUN
-#define FUNGEHANDPRINT 0x4346554e
+#ifndef FUNGEHANDPRINT
+#  define FUNGEHANDPRINT 0x4346554e
+#endif
 /// Version, for -V.
 #define APPVERSION  "0.2.1-pre2"
 /// For y instruction.

--- src/funge-space/funge-space.c	2008-07-06 00:49:23.000000000 +0100
+++ src/funge-space/funge-space.c	2008-07-06 18:09:22.000000000 +0100
@@ -312,7 +312,7 @@
 	// Row in fungespace
 	FUNGEVECTORTYPE y = 0;
 	FUNGEVECTORTYPE x = 0;
-	size_t linelen = strlen(program) + 1;
+	size_t linelen = strlen(program);
 
 	for (size_t i = 0; i < linelen; i++) {
 		switch (program[i]) {
